### ISSUE TITLE
Gravatar URL stored in $context['user'] incorrect

### ISSUE THREAD

**Issue by Ant59** [elkarte/Elkarte#1662]
In the user section of the $context array, if using Gravatars, the URL is constructed incorrectly.

Using a Gravatar results in $context['user']['avatar']['href'] becoming prepended with the avatar directory before the Gravatar URL, such as:

`http://beta.xydre.com/avatars///www.gravatar.com/avatar/6a18357cc2cf108b8a3a26d811a8ad91d=65&amp;r=pg`

This is due to line 2608 in Subs.php using a conditional to check for "gravatar" in the $user_info['avatar']['url'] variable. However, this variable is never populated with "gravatar", instead housing the actual gravatar.com URL itself already.

https://github.com/elkarte/Elkarte/blob/0359020e47c434630b672e0d8471e5ec7dc547d4/sources/Subs.php#L2608

I think that the line in Subs.php should be looking at $user_info['avatar']['name'], since *this is* set to "gravatar".

Output of $user_info['avatar'] array:
```
[avatar] => Array
(
    [url] => //www.gravatar.com/avatar/6a18357cc2cf108b8a3a26d811a8ad91d=65&r=pg
    [filename] => 
    [custom_dir] => 
    [id_attach] => 0
    [name] => gravatar
    [image] => 
    [href] => //www.gravatar.com/avatar/6a18357cc2cf108b8a3a26d811a8ad91d=65&r=pg
    [gravatar_preview] => //www.gravatar.com/avatar/6a18357cc2cf108b8a3a26d811a8ad91d=65&r=pg
)
```

Output of $context['user']['avatar'] array:
```
[avatar] => Array
(
    [href] => http://beta.xydre.com/avatars///www.gravatar.com/avatar/6a18357cc2cf108b8a3a26d811a8ad91d=65&amp;r=pg
    [image] => 
)
```

**Comment by emanuele45** [comment]
To check, but I think this has been fixed in the #1360 or other related.

**Comment by emanuele45** [comment]
Yep, looks like it was "fixed" with the 1.0.3 changes.
@Ant59 if you see it again feel free to reopen. ;D
